src/generated/graphql.tsx

```diff
@@ -4,206 +4,223 @@ export type Maybe<T> = T | null;
 export type RequireFields<T, K extends keyof T> = { [X in Exclude<keyof T, K>]?: T[X] } & { [P in K]-?: NonNullable<T[P]> };
 /** All built-in and custom scalars, mapped to their actual values */
 export type Scalars = {
-    ID: string;
```

Comment:
even with wes' editor config this won't format correctly for me so I keep getting these diffs too... I've been going in manually and moving to 4 spaces with a replace-all... but maybe we should just leave it as a 2 space for this file.

Reply:
yeah, i don't think we should worry too much about this since it's a generated file. my emacs setup is a _little_ overzealous about correcting formatting if i do anything in a file though, so i think that's why it changed on my end.